.env.sample

```diff
@@ -1,22 +0,0 @@
-APP_NAME=
-AWS_ACCESS_KEY_ID=
-AWS_SECRET_ACCESS_KEY=
-AWS_REGION=
-CW_FILES_PREFIX=
-CW_API=
-DEV_USER_ID=
-DEV_USER_TOKEN=
-ESP_API=
-GFW_API=
-GOOGLE_ANALYTICS_ID=
-USER_REPORT_KEY=
-IPSTACK_GEOLOCATION_API_KEY=
-POSTGRES_URL=
-S3_BUCKET_NAME=
-USER_SURVEY_SPREADSHEET_URL=
-IPSTACK_GEOLOCATION_API_KEY=
-
-
-FEATURE_AGRICULTURE=
-
-
```

Comment:
please, bring back the `.env.sample` file, you can modify it by adding the feature flag you added `FEATURE_NDCS_ENHANCEMENTS`

Reply:
Will restore - this was removed in error.